sql/rpl_slave.cc

```diff
@@ -4937,19 +4938,18 @@ static int exec_relay_log_event(THD *thd, Relay_log_info *rli,
     if (ev->get_type_code() == binary_log::FORMAT_DESCRIPTION_EVENT &&
         ev->server_id != ::server_id && ev->common_header->log_pos != 0 &&
         rli->is_parallel_exec() && rli->curr_group_seen_gtid) {
-      if (coord_handle_partial_binlogged_transaction(rli, ev))
-      /*
```

Comment:
The PXC source without WITH_WSREP should be identical to the PS source, except for whitespace differences. I will add a script / CI test soon that checks/enforces this (and also fix existing differences).

Moving the comment, and adding the `{` to the if line breaks this rule. A possible fix is to:

 * keep the comment where it is
* add the `{` as a separate line into the existing WITH_WSREP block
* add a separate ifdef WITH_WSREP around the closing `}`

Reply:
Hi Zsolt, Thank you for reviewing the changes. There is a duplication of comment in PS and PXC. Do you suggest to keep them as is?

Reply:
Yes, if it is duplicated in PS too, we should keep it duplicated.